docs/sce_file_contents.md

```diff
@@ -0,0 +1,131 @@
+
+
+### Gene information and metrics
+
+Gene information and metrics calculated from the RNA-seq expression data are stored as a `DataFrame` in the `rowData` slot, with the Ensembl ID as the names of the rows.
+
+```r
+gene_info <- rowData(sce)
+```
+
+The following columns are included for all genes. 
+Metrics were calculated using the [`scuttle::addPerFeatureQCMetrics`](https://rdrr.io/github/LTLA/scuttle/man/addPerFeatureQCMetrics.html) function.
+
+| Column name   | Contents                                                         |
+| ------------- | ---------------------------------------------------------------- |
+| `gene_symbol` | [HUGO](https://www.genenames.org) gene symbol, if defined        |
+| `mean`        | Mean count across all cells/droplets                             |
+| `detected`    | Number of cells in which the gene was detected (gene count > 0 ) |
+
+### Experiment metadata
+
+Metadata associated with data processing is included in the `metadata` slot as a list.
+
+```r
+expt_metadata <- metadata(sce)
+```
+
+| Item name           | Contents                                                                                                 |
+| ------------------- | -------------------------------------------------------------------------------------------------------- |
+| `salmon_version`    | Version of `salmon` used for initial mapping                                                             |
+| `reference_index`   | Transcriptome reference file used for mapping                                                            |
+| `total_reads`       | Total number of reads processed by `salmon`                                                              |
+| `mapped_reads`      | Number of reads successfully mapped                                                                      |
+| `mapping_tool`      | Pipeline used for mapping and quantification (`alevin-fry` for all current data in ScPCA)                |
+| `alevinfry_version` | Version of `alevin-fry` used for mapping and quantification                                              |
+| `af_permit_type`    | `alevin-fry generate-permit-list` method used for filtering cell barcodes                                |
+| `af_resolution`     | `alevin-fry quant` resolution mode used                                                                  |
+| `usa_mode`          | Boolean indicating whether quantification was done using `alevin-fry` USA mode                           |
+| `af_num_cells`      | Number of cells reported by `alevin-fry`                                                                 |
+| `transcript_type`   | Transcripts included in gene counts: `spliced` for single-cell samples and `unspliced` for single-nuclei |
+
+## Additional fields for filtered cells
+
+After {ref}`filtering for empty droplets <processing_information:Filtering cells>`, some additional fields are added to the `SingleCellExperiment` object.
```

Comment:
```suggestion
After {ref}`filtering and removing empty droplets <processing_information:Filtering cells>`, some additional fields are added to the `SingleCellExperiment` object.
```
I don't know if this is the best way to fix this but `filtering for empty droplets` made it sound like we were selecting for those to me, so I'm trying this to see if it helps clarify it.